drivers/bridge/bridge_store.go

```diff
@@ -183,6 +184,10 @@ func (ncfg *networkConfiguration) UnmarshalJSON(b []byte) error {
 		ncfg.ContainerIfacePrefix = v.(string)
 	}
 
+	if v, ok := nMap["HostIP"]; ok {
```

Comment:
Why add this if statement for `HostIP` but not for `DefaultBindingIP`, `DefaultGatewayIPv4` and `DefaultGatewayIPv6`?

Reply:
its a safer check, to be certain that the key `HostIP` exists

Reply:
Alright, thanks for the clarification.